source/agora/common/Config.d

```diff
@@ -560,3 +574,79 @@ unittest
     auto scp_quorum = toSCPQuorumSet(quorum);
     assert(scp_quorum.toQuorumConfig() == quorum);
 }
+
+/*******************************************************************************
+
+    Parse the `event_handlers` config section
+
+    Params:
+        ptr = pointer to the Yaml node containing the loggers configuration
+        c = the parsed command line arguments, for override
+
+    Returns:
+        the parsed event handlers
+
+*******************************************************************************/
+
+private EventHandlerConfig parserEventHandlers (Node* node, const ref CommandLine c)
```

Comment:
Why are you parsing the CommandLine and then not using it?

Reply:
No, I am just saying that it should not be a parameter.

Reply:
However you have already included it in the latest change, so it's ok to keep it.